### DISCUSSION TITLE
How to write latex in mm

### DISCUSSION THREAD

**Discussion by jasonppy** [mmistakes/minimal-mistakes#2896] [Q&A]
## Motivation

I need to write some mathematics in my post, which is why I need latex

I basically haven't found the way to write latex in mm. I tried some tutorials online (like [this one](https://www.janmeppe.com/blog/How-to-add-mathjax-to-minimal-mistakes/)) but none of them works.

## Suggestion

I forked [mm-starter](https://github.com/mmistakes/mm-github-pages-starter) repo and start blogging. I wonder if you can tell me how to add the latex functionality to it.

**Comment by hzzg0727** [comment]
Hello. I met the same problem, and my solution is as follows.
In the blog directory, create _includes/head/custom.html, as shown below.

![image](https://user-images.githubusercontent.com/62739713/197331009-e1d60916-cdfc-4ffc-9c1b-6523775f3948.png)

And then, add the following content into custom.html:

```html
<script src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script type="text/x-mathjax-config">
    MathJax.Hub.Config({
      tex2jax: {
        skipTags: ['script', 'noscript', 'style', 'textarea', 'pre'],
        inlineMath: [['$','$']]
      }
    });
</script>
```
Done!
I hope this will help you.
